userspace/libsinsp/parsers.cpp

```diff
@@ -569,12 +564,15 @@ void sinsp_parser::store_event(sinsp_evt &evt) {
 	// Copy the data
 	//
 	auto tinfo = evt.get_tinfo();
+	uint8_t *last_event_data = tinfo->get_last_event_data();
+	if(last_event_data != nullptr) {
+		free(last_event_data);
+	}
```

Comment:
In case an old event_data was already present, free it and override it with new event data.
This might happens in case we drop an event, eg:
* `open_e` calls store_evt
* `open_x` is dropped
* another `open_e` calls store_evt